types/readable-stream/tsconfig.json

```diff
@@ -12,6 +12,11 @@
         "typeRoots": [
             "../"
         ],
+        "paths": {
```

Comment:
Does this work? How does this effect the final @types/readable-stream NPM package? Are there any issues to expect if someone uses readable-stream togehter with node 12 definitions?
By the way, maybe it's better to skip stream changes in this PR to get the other changes merged and then add one more PR for stream.

Reply:
Forgot about this bit :)